app/web/template/choose_attorneys_summary.gohtml

```diff
@@ -22,7 +22,7 @@
                     </a>
                 </dd>
                 <dd class="govuk-summary-list__actions">
-                    <a class="govuk-link govuk-body-s" href="#">
+                    <a class="govuk-link govuk-body-s" href="{{ $.RemoveAttorneyPath }}?id={{ .ID }}">
```

Comment:
this will need to be `{{ link $.App $.RemoveAttorneyPath }}` to work in welsh

Reply:
also for the other `href`s in here

Reply:
Hmmmm maybe we should start adding in welsh integration tests to catch this type of thing?